marklogic-unit-test-modules/src/main/ml-modules/root/test/test-helper.xqy

```diff
@@ -44,6 +44,14 @@ declare variable $test:__LINE__ as xs:int :=
 
 declare variable $test:__CALLER_FILE__ := test:get-caller();
 
+declare function test:get-test-data-path($caller-path as xs:string) as xs:string
```

Comment:
Seeing this now, I am thinking instead of my proposed function name, it should be "get-test-data-directory-uri", because that's what it's returning - the directory URI containing the test-data files for this test.

Can you also add a function comment that uses the above description - i.e. "Returns a URI identifying the directory that contains any test data files associated with the test module that invokes this."